.github/workflows/publish.yml

```diff
@@ -33,13 +33,10 @@ jobs:
              # we need to build our own version of cross-enabled linuxkit, while upstream is sorting things out
              make LINUXKIT_VERSION=master LINUXKIT_SOURCE=https://github.com/rvs/linuxkit.git linuxkit
              # constraining environment for riscv64 builds
-             EVE_BUILDER_IMAGE=$(./build-tools/bin/linuxkit pkg show-tag pkg/alpine)-riscv64
-             echo "PKGS=pkg/alpine pkg/ipxe pkg/mkconf pkg/mkimage-iso-efi pkg/mkimage-raw-efi" >> "$GITHUB_ENV"
+             echo "PKGS=pkg/alpine pkg/ipxe pkg/mkconf pkg/mkimage-iso-efi pkg/mkimage-raw-efi pkg/uefi pkg/u-boot pkg/grub pkg/new-kernel" >> "$GITHUB_ENV"
```

Comment:
Can we abstract these `PKGS`? Maybe have a `pkgs.<arch>` file in the root directory, and the `Makefile`, if it finds that, uses it, else it reads per its current logic?

Reply:
Yeah, one other way of doing it is via Makefile itself -- do you think it'll be a better approach @deitch ?

Reply:
Either way is fine. I prefer an explicit file, as it concentrates changes in one place. If we want to change packages for, say, aarch64, we aren't trying to figure out a `Makefile`, just a simple file with one package per row.

Reply:
Just moved it to Makefile @deitch